### DISCUSSION TITLE
Please notify to Slack when pull request was updated

### DISCUSSION THREAD

**Discussion by quietnumeric** [community/community#22242] [New to GitHub]
<p>I’m using Slack notification plugin.<br>
When pull request (has been sent notification to Slack) was updated (editing title or comment, adding new commit), plugin send nothing to Slack.<br>
I hope plugin edit existing Slack message when pull request title or comment were updated (like issues and reviews and comments were edited).<br>
I hope plugin send new Slack message when new commit was added to pull request.<br>
Colleague wants to know not only content of commit but addition of commit itself.</p>

**Comment by quietnumeric** [comment]
<p>Does anyone care?<br>
I pray that my voice will reach GitHub developers.</p>

**Comment by darkRaspberry** [comment, marked as answer]
<p>As I can understand you are using webhocks.</p>
<p>Lets see how webhocks works:<br>
When a event occurs the webhocks send a trigger to slack, And slack render it to show you a notification.<br>
(According to GitHub- Webhooks allow external services to be notified when certain events happen. When the specified events happen, we’ll send a POST request to each of the URLs you provide. Learn more in our <a href="https://docs.github.com/webhooks/" rel="noopener nofollow ugc">Webhooks Guide</a>.)</p>
<p>The same happen when you create a Pull request the webhocks send a trigger to slack and slack render it but, when you update the pull request the webhocks don’t send any triggers, so the slack can’t render edits.</p>
<p>What u can do:</p>
<ul>
<li>You might want to create a issue with the slack plugin, and tell their developer about this issue.</li>
<li>You are using GitHub means you a developer then you create your own webhocks which can do your desired work.</li>
<li>Or use any other slack plugin which do this work.</li>
</ul>
<p>(Side HINT- Try checking your config May be you have option to do this but it was turned off.)</p>

**Comment by quietnumeric** [comment]
<p>Thank you so.</p>
<p>I have already known webhook is sendable all about pull request.<br>
At repo/Settings &gt; Webhooks &gt; Add webhook.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/d/4d61a0dcd412a35a9357d64624a7727e6015f67d.png" data-download-href="/uploads/short-url/b2xVQBXVtYYqUS7nOo5srgHvzqJ.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11306822/181069700-f0a8a57e-cd15-4442-8438-6f3fd214e49b.png" alt="image" data-base62-sha1="b2xVQBXVtYYqUS7nOo5srgHvzqJ" width="690" height="220" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4d61a0dcd412a35a9357d64624a7727e6015f67d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">806×257 14.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
And I tried recieve events at my server app then it was succeeded.
<p>But i didn’t know slack plugin project (I’m embarrassed).<br>
I’ll request that to there:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/integrations/slack" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11306822/181069702-baa1414f-3f78-4451-abc9-de5d8314d64c.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/integrations/slack" target="_blank" rel="noopener nofollow ugc">integrations/slack</a></h3>

<p>Bring your code to the conversations you care about with the GitHub and Slack integration - integrations/slack</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
